miner/worker.go

```diff
@@ -247,6 +254,16 @@ func (w *worker) setRecommitInterval(interval time.Duration) {
 	w.resubmitIntervalCh <- interval
 }
 
+// disablePreseal disables pre-sealing mining feature
+func (w *worker) disablePreseal() {
+	w.noempty = true
+}
+
+// enablePreseal enables pre-sealing mining feature
+func (w *worker) enablePreseal() {
+	w.noempty = false
```

Comment:
Lets convert these and the reads to atomics, just to make it properly correct. Otherwise LGTM

Reply:
Fixed